app/subscriber/src/App.tsx

```diff
@@ -46,7 +46,16 @@ function App() {
         </LayoutAnonymous>
       </Show>
       <ToastContainer />
-      <Tooltip variant="light" id="main-tooltip" place="top" />
+      <Tooltip
+        style={{
+          backgroundColor: '#FFFFCC',
```

Comment:
In line style unfortunately as per the docs suggested. Couldn't access from styled component.